scene/gui/tree.h

```diff
@@ -218,6 +218,15 @@ class TreeItem : public Object {
 	bool is_checked(int p_column) const;
 	bool is_indeterminate(int p_column) const;
 
+	void propagate_check(int p_column);
+	TypedArray<TreeItem> propagate_check_and_get_affected(int p_column);
+
+private:
+	/* check helper */
```

Comment:
```suggestion
	// Check helpers.
```
That's the preferred comment style in codebase.